masq_lib/src/utils.rs

```diff
@@ -127,6 +129,39 @@ pub fn exit_process_with_sigterm(message: &str) {
     }
 }
 
+pub trait ExpectDecent<T> {
+    fn expect_decent(self, msg: &str) -> T;
+}
+
+impl<T> ExpectDecent<T> for Option<T> {
+    fn expect_decent(self, subject: &str) -> T {
```

Comment:
Rename this to expect_value() which better express the aim of this utility. 

And add #[track_caller] which is an attribute thanks to which we don't dive in the guts of this fn and if informed of a line or something then its the line where expect_value() is